build-logic/publication/src/main/kotlin/convention.publish-android-library.gradle.kts

```diff
@@ -17,13 +17,14 @@ configure<LibraryExtension> {
     val allVariantNames = mutableListOf<String>()
     var registeredVariants = 0
 
+    // todo use new publishing: https://developer.android.com/studio/releases/gradle-plugin#build-variant-publishing
     libraryVariants
         .matching {
             allVariantNames += it.name
             it.name == publishExtension.variant.get()
         }
         .whenObjectAdded {
-            publishing {
```

Comment:
android library now have it's own publishing extension: https://developer.android.com/studio/releases/gradle-plugin#build-variant-publishing